pkg/apis/config/feature_flags.go

```diff
@@ -207,15 +232,15 @@ func NewFeatureFlagsFromMap(cfgMap map[string]string) (*FeatureFlags, error) {
 	if tc.EnableAPIFields == AlphaAPIFields {
 		tc.EnableTektonOCIBundles = true
 		tc.EnableCustomTasks = true
-		tc.EnableSpire = true
+		tc.EnforceNonfalsifiability = EnforceNonfalsifiabilityWithSpire
```

Comment:
does this mean the spire feature is enabled by default for alpha api flag?

Reply:
yes.